README.md

```diff
@@ -12,144 +12,361 @@
     </div>
 </div>
 
-## Workflow is a Golang workflow framework that encompasses these main features:
-- Defining small units of work called "Steps"
-- Consumer management and graceful shutdown
-- Supports event streaming platforms such as Kafka and Reflex (or you can write your own implementation of the EventStreamer interface!)
-- Built in support for timeout operations (e.g. account cool down periods etc).
-- Built in support for callbacks (e.g. Call an async endpoint and trigger the callback from a webhook handler).
-- Connect two workflows together. Wait for specific events from another workflow and make it part of your workflow!
-- Super Duper testable
-
+# Workflow
+ Workflow is an event driven workflow that allows for robust, durable, and scalable sequential business logic to
+ be executed in a deterministic manner.
```

Comment:
Looks like this is being rendered as a heading, is that intentional?

Reply:
Hm strange it wasnt on my machine 🤔

Reply:
This is the rendering I get

<img width="1069" alt="Screenshot 2024-08-13 at 16 05 08" src="https://github.com/user-attachments/assets/63244bb8-6745-4866-8f89-5434724ef642">

Reply:
I'm looking in the browser at https://github.com/luno/workflow/tree/andreww-updateReadMe, is that what you're looking at?

Reply:
I think removing the space before, or maybe also adding a line in between the two, should fix

```suggestion
Workflow is an event driven workflow that allows for robust, durable, and scalable sequential business logic to
be executed in a deterministic manner.
```